modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -3,31 +3,30 @@
 
 This section contains information about registering Salt clients running {ubuntu} operating systems.
 
-{susemgr} supports {ubuntu} 16.04 LTS and 18.04 LTS Clients using Salt.
+{productname} supports {ubuntu} 16.04 LTS and 18.04 LTS Clients using Salt.
 Traditional clients are not supported.
 
 [IMPORTANT]
 ====
-Canonical does not endorse or support {susemgr}.
+Canonical does not endorse or support {productname}.
 ====
 
 Bootstrapping is supported for starting {ubuntu} clients and performing initial state runs such as setting repositories and performing profile updates.
 However, the root user on {ubuntu} is disabled by default, so to use bootstrapping, you will require an existing user with [command]``sudo`` privileges for Python.
 
 
-
 == Prepare to Register
 
 Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
 
-
-
 ifeval::[{suma-content} == true]
 
 .Procedure: Adding client tools channels
```

Comment:
Let's split this into two, one for the webui, and one for the CLI. Then we can move L28 up to before both of them.

Reply:
ditto